ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -54,20 +54,35 @@ class BottomCommandingDemoController: UIViewController {
         CommandingItem(title: "Boolean Item " + String($0), image: homeImage, action: commandAction, isToggleable: true)
     }
 
-    private lazy var expandedListSections: [CommandingSection] = [
+    private lazy var fewExpandedListSections: [CommandingSection] = [
```

Comment:
not crazy about the name. smallerSet vs largerSet? shortList vs longlist?

Reply:
Haha you're totally right, this is awful.

Reply:
I updated the naming.